src/components/Booklist.js

```diff
@@ -1,19 +1,24 @@
 import { useSelector } from 'react-redux';
+import { useEffect } from 'react';
 import Book from './Book';
 import CreateBook from './create';
+import { fetchBooks } from '../redux/books/books';
 
 const Books = () => {
   const books = useSelector((state) => state.booksReducer);
+  useEffect(() => {
+    fetchBooks();
+  }, [books]);
 
   return (
     <div className="book-list">
       <ul className="book">
         {books.map((book) => (
           <Book
-            key={book.id}
-            id={book.id}
-            title={book.title}
-            author={book.author}
+            key={book.item_id}
+            id={book.item_id}
+            title={book.title.title}
+            author={book.title.author}
             category={book.category}
           />
         ))}
```

Comment:
- [ ] it seems the configuration of redux is good but when I run your application  at  `localhost:3000` I see nothing please make sure that your app is working properly 
<img width="769" alt="Screenshot 2023-01-02 at 15 17 54" src="https://user-images.githubusercontent.com/61790587/210243738-757f6960-cf12-4a5e-b345-5142176588b4.png">

Reply:
I tried but i cant see my app in the local could you help plz https://github.com/youmari